frontstage/views/surveys/help/technical_help.py

```diff
@@ -55,6 +58,7 @@ def send_help_technical_message(session):
     """Sends secure message for the help pages"""
     form = SecureMessagingForm(request.form)
     option = request.args.get("option", None)
+    logger.info("Request: " + str(request))
```

Comment:
As above

Reply:
Nope. Accidentally added for test purposes.